### PR TITLE
Add service instance to iOS SDK [21916]

### PR DESCRIPTION
**Pull request by lqgioan**
## Description
- Added code to show the Service Instance feature.

## Motivation & Context
Ticket: https://app.shortcut.com/credify/story/21916/feature-ios-add-service-instance-to-ios-sdk

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: Credify/Credify/Objects/Entities.swift [hunk 1]
```diff
@@ -251,3 +251,25 @@ enum OnboardingStatus : String {
     case canceled = "CANCELED"
     case failed = "FAILED"
 }
+
+/// 20771: Add Service Instance
+public enum ProductType : String {
+    // Insurance
+    case insurance = "insurance"
```
Comment (ngo275) [inline Credify/Credify/Objects/Entities.swift:258]: @lqgioan Please comment out products that are unavailable at the moment. So please enable only `insurance` series this month

File: Credify/Credify/WebView/WebPresenter.swift [hunk 2]
```diff
@@ -86,6 +91,18 @@ class WebPresenter: WebPresenterProtocol {
                     ]
                 )
             }
+            if case let .serviceInstance(user, _, _) = context {
+                doPostMessage(
+                    webView,
+                    type: ACTION_TYPE,
+                    action: SendMessageHandler.actionLogin.rawValue,
+                    payload: [
```
Comment (ngo275) [inline Credify/Credify/WebView/WebPresenter.swift:99]: This doesn't use base64?

File: ExampleApp/ExampleApp/ViewController.swift [hunk 3]
```diff
@@ -79,4 +82,12 @@ class ViewController: UIViewController {
         }
     }
     
+    @IBAction func showServiceInstanceClick(_ sender: Any) {
+        serviceX.Passport().showDetail(
+            from: self,
+            user: user,
+            marketId: "8af0e885-a06c-4508-8d17-03e4fa1ea526",
```
Comment (ngo275) [inline ExampleApp/ExampleApp/ViewController.swift]: Please move the hard-coded value to the beginning of the file along with `API_KEY`, etc..

Comment (lqgioan) [reply ExampleApp/ExampleApp/ViewController.swift]: I fixed it

File: Credify/Credify/Objects/Entities.swift (hunk 1 above)
Comment (lqgioan) [reply Credify/Credify/Objects/Entities.swift:258]: I comment all products except `insurance` series

File: Credify/Credify/WebView/WebPresenter.swift (hunk 2 above)
Comment (lqgioan) [reply Credify/Credify/WebView/WebPresenter.swift:99]: @ngo275 I only use base64 for the payload

**Review by ngo275 — APPROVED** [review]

**Pull request merged**